src/pages/wallet/send.vue

```diff
@@ -432,8 +432,10 @@ export default {
             message: "Getting transaction information",
             sending: true
           });
+          const isSweepAll = this.newTx.amount === this.unlocked_balance / 1e9;
```

Comment:
```suggestion
          const isSweepAll = this.newTx.amount === this.unlocked_balance / 1e9;
```
shouldn't we do this in `transfer`? The UI should only care about sending amounts and not worry about wether to make a sweep all or a normal send

Reply:
true, can do that now this decision has been made